lms/static/styles/components/_FileList.scss

```diff
@@ -38,7 +40,7 @@ $file-list-icon-size: 24px;
   &.is-selected {
     // nb. Argument must be passed to `invert` even though "1" is the default
     // to work around https://bugs.chromium.org/p/chromium/issues/detail?id=964696.
-    filter: #{"invert(1)"}; // Quoted to avoid conflict with SASS mixin.
+    filter: #{'invert(1)'}; // Quoted to avoid conflict with SASS mixin.
```

Comment:
Unintentional.

Because .prettierrc has `"singleQuote": true` it changed the quote style when saving the file.

Is this something I should avoid in the future?

Reply:
> Because .prettierrc has "singleQuote": true it changed the quote style when saving the file.

Ah. In the lms repo prettier currently only applies to JavaScript. See the `format` command in `package.json`. We already use Prettier to format .scss files in the client repo so it would make sense to use it here as well. See `package.json` in the client repo. If we're going to make that change it should be in a new PR which modifies the "format" and "checkformatting" scripts and also reformats the existing `.scss` files.